extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/conf/CASGuacamoleProperties.java

```diff
@@ -68,5 +69,30 @@ private CASGuacamoleProperties() {}
         public String getName() { return "cas-clearpass-key"; }
 
     };
+  
+   /**
+    * The attribute used for group membership
+    * example:  memberOf  (case sensitive)
+    */
+    public static final StringGuacamoleProperty CAS_GROUP_ATTRIBUTE =
+         new StringGuacamoleProperty() {
 
+    @Override
+    public String getName() { return "cas-group-attribute"; }
+
+    };
+
+   /**
+    * The attribute used for group DN Format
+    * such as CN=%s,OU=myou,DC=example,DC=com  (case sensitive)
+    * used to strip all but %s.  This is only necessary when
+    * CAS backend is LDAP. 
+    */
```

Comment:
Are there other cases where CAS might have group names in a specific format? If so, perhaps this should be more generic and not specific to an LDAP DN?

Reply:
CAS groups come from LDAP, databases, and Grouper.  AFAIK LDAP is the only place you have the notion of DNs with potentially irrelevant RDNs that you *may* want to strip.  Database entries are usually singular group names, and grouper entries are fully significant.  Do you know of any other cases?